### ISSUE TITLE
[Feature] Add utility scripts to convert between cardano-cli text envelopes and open ssl .pem files

### ISSUE THREAD

**Issue by Ryun1** [IntersectMBO/credential-manager#6]
- it would be nice to include the ability to convert between cardano-cli text envelope keys and open ssl .pem representations

**Comment by yushih** [comment]
@Ryun1 we had the same problem so I developed this Bash script to convert from Cardano private key file to x.509 pem:
`cardano-skey-to-x509.sh`: 
```
if [ -z "$1" ]; then
    echo "cardano-skey-to-x509.sh skey-file"
    exit
fi

if [ ! -f $1 ]; then
    echo "skey-file \"$1\" doesn't exist"
    exit
fi

if ! grep '"type": "PaymentSigningKeyShelley_ed25519"' $1 > /dev/null; then
    echo "skey-file \"$1\" is not Cardano private key"
    exit
fi

privKeyHex=`sed -n -e  's/^.*"cborHex": "5820\(.*\)"/\1/p' $1` #trim cbor header
if [ ${#privKeyHex} -ne 64 ]; then
    echo "private key format error"
    exit
fi

asn1header="30:2e:02:01:00:30:05:06:03:2b:65:70:04:22:04:20"

echo "${asn1header}${privKeyHex}" | xxd -r -ps | openssl pkey -outform pem 
```

**Comment by Ryun1** [comment]
hey @yushih 
This is super helpful
Thank you!

**Comment by jhbertra** [comment]
@yushih @Ryun1 I have just added a new `orchestrator-cli` utility that does the conversion. Usage:

```bash
# Converts a pem file into a text envelope signing key file
$ orchestrator-cli from-pem my-openssl-private-key.pem output.skey
# Converts a text envelope signing key file into an open ssl pem file
$ orchestrator-cli from-pem my-key.skey output.pem
```

**Comment by jhbertra** [comment]
Fixed by Commit 17f12576e1f7cd78039da88eb154f7841e9cd378

**Comment by jhbertra** [comment]
Fixed by Commit 17f12576e1f7cd78039da88eb154f7841e9cd378

**Comment by Ryun1** [comment]
Awesome, thank you!
